pkg/ingress/controller/apisix_route.go

```diff
@@ -188,15 +188,11 @@ func (c *apisixRouteController) handleSyncErr(obj interface{}, err error) {
 		c.workqueue.Forget(obj)
 		return
 	}
-	if c.workqueue.NumRequeues(obj) < _maxRetries {
```

Comment:
See the use pattern in https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/endpoint/endpoints_controller.go#L356.

We just reference it.

Reply:
I suggest to cancel the maximum number of times limit first, and add it when our admission and crd status are improved. Now we need real-time error reporting to quickly solve the problem.

Reply:
I cannot claim a distinct point about this now. Just merge it if it makes some trouble at present.

Reply:
OK